docs/developer-docs/integrations/ethereum/cketh.mdx

```diff
@@ -0,0 +1,104 @@
+| [Minter](https://github.com/dfinity/ic/tree/master/rs/ethereum/cketh/minter)                        | [sv3dd-oaaaa-aaaar-qacoa-cai](https://dashboard.internetcomputer.org/canister/sv3dd-oaaaa-aaaar-qacoa-cai) ([Dashboard](https://sv3dd-oaaaa-aaaar-qacoa-cai.raw.icp0.io/dashboard)) | [jzenf-aiaaa-aaaar-qaa7q-cai](https://dashboard.internetcomputer.org/canister/jzenf-aiaaa-aaaar-qaa7q-cai) ([Dashboard](https://jzenf-aiaaa-aaaar-qaa7q-cai.raw.ic0.app/dashboard)) |
+| [Ledger](https://github.com/dfinity/ic/tree/master/rs/rosetta-api/icrc1/ledger)                     | [ss2fx-dyaaa-aaaar-qacoq-cai](https://dashboard.internetcomputer.org/canister/ss2fx-dyaaa-aaaar-qacoq-cai)                                                                          | [apia6-jaaaa-aaaar-qabma-cai](https://dashboard.internetcomputer.org/canister/apia6-jaaaa-aaaar-qabma-cai)                                                                          |
+| [Index](https://github.com/dfinity/ic/tree/master/rs/rosetta-api/icrc1/index-ng)                    | [s3zol-vqaaa-aaaar-qacpa-cai](https://dashboard.internetcomputer.org/canister/s3zol-vqaaa-aaaar-qacpa-cai)                                                                          | [sh5u2-cqaaa-aaaar-qacna-cai](https://dashboard.internetcomputer.org/canister/sh5u2-cqaaa-aaaar-qacna-cai)                                                                          |
+| [Helper]([https://github.com/dfinity/ic/blob/master/rs/ethereum/cketh/minter/ERC20DepositHelper.sol) | [0x7574eB42cA208A4f6960ECCAfDF186D627dCC175](https://etherscan.io/address/0x7574eB42cA208A4f6960ECCAfDF186D627dCC175](https://github.com/dfinity/ic/blob/master/rs/ethereum/cketh/minter/EthDepositHelper.sol))                                                               | [0xb44B5e756A894775FC32EDdf3314Bb1B1944dC34](https://sepolia.etherscan.io/address/0xb44B5e756A894775FC32EDdf3314Bb1B1944dC34)                                                       |
+
+
+The **ckETH minter** is responsible for the minting and burning of ckETH tokens.
+It uses the following workflow:
+
+- Tokens are minted when a user calls the `deposit` function of the helper
+  contract on Ethereum with some ETH attached. The deposit function takes the
+  receiver's principal as an argument and emits a `ReceivedEth` event with the
+  `sender`, `value`, and `receiver` as payload.
+- The ckETH minter listens to the `ReceivedEth` event by periodically fetching
+  logs from multiple Ethereum JSON RPC cloud providers and mints the
+  corresponding amount of ckETH to the receiver's account on the ckETH ledger.
+- Users can withdraw by creating an ICRC-2 approval on the ledger and calling `withdraw_eth` with a receiver ETH address as an argument on the minter.
+- The minter will then burn the ckETH on the ledger and create an Ethereum transaction and submit it to the Ethereum network.
```

Comment:
```suggestion
- The minter will then burn the ckETH on the ledger and create an Ethereum transaction that is submitted it to the Ethereum network.
```

Reply:
Why do you want to have the 2nd part of the sentence passive when the first part is active?